### ISSUE TITLE
Average length of pretraining protein sequences

### ISSUE THREAD

**Issue by Yijia-Xiao** [facebookresearch/esm#173]
Hi!
Thanks for the great work. I have a question regarding _[MSA Transformer](https://www.biorxiv.org/content/10.1101/2021.02.12.430858v1)_. The paper provided the distribution of **_MSA depths_** of the training set. However, the length distribution of protein is not provided.

So I am wondering whether it is possible to disclose some statistics about the **_length distribution_**?

Thanks!

Best,
Yijia Xiao

**Comment by tomsercu** [comment]
The lengths of the proteins are straightforwardly found in the uniref50 database, which sequences were used as seed seqs to construct the MSAs. We used this version, quite outdated now but I would guess the length statistics won't change so much: https://ftp.uniprot.org/pub/databases/uniprot/previous_releases/release-2018_03/uniref/

**Comment by Yijia-Xiao** [comment]
Got it, thank you :-) @tomsercu
